ci/jenkins/jobs/default.yaml

```diff
@@ -0,0 +1,7 @@
+- defaults:
```

Comment:
I dont see this file in antrea repo? Can you see what its done there and make the corresponding change here too